physical/raft/raft.go

```diff
@@ -445,6 +445,10 @@ func NewRaftBackend(conf map[string]string, logger log.Logger) (physical.Backend
 		updateInterval = interval
 	}
 
+	if reconcileInterval != 0 && updateInterval != 0 && reconcileInterval < updateInterval {
```

Comment:
What if they only specify one, but the default for the other is inconsistent with the specified value?

Reply:
https://github.com/hashicorp/vault/pull/15521/commits/75677ea0567f1540367c772451453dc3fa4ed098